packages/components/src/styles/globals.scss

```
@@ -1,5 +1,29 @@
+@font-face {
+  font-family: "MW-Flow-Regular";
+  src: url(../fonts/Inter-Regular.woff2);
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "MW-Flow-Regular";
+  src: url(../fonts/Inter-Bold.woff2);
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: "MW-Flow-PageTitle";
```

Vielleicht sollten wir die Fonts nach ihrer eigentlichen Family benennen. Ich finde hier schon zu sagen, das ist die PageTitle Font falsch.

```suggestion
  font-family: "ProximaNovaBold";
```